Documentation/devel/stage1-implementors-guide.md

```diff
@@ -118,3 +118,22 @@ Examples
     ]
 }
 ```
+
+Filesystem Layout Assumptions
+-----------------------------
+
+### opt/stage2
+
+As mentioned above, in rkt, the apps will be executed during stage2.
+rkt expects the directory path `opt/stage2` to exist under the stage1 rootfs.
```

Comment:
@iaguis @krnowak OK, then can we create `/rkt/status` `/rkt/env` in stage0 as well?
The error @steveeJ got is from
https://github.com/coreos/rkt/blob/master/stage0/run.go#L641

Seems it's because `/rkt` directory does not exist.